frontend/src/features/SidebarSuite/uiSettings/MultiLingualSelector.tsx

```diff
@@ -0,0 +1,167 @@
+// import * as React from "react";
```

Comment:
is this a surprise file that will help us later? :)

Reply:
:laughing: It is the legacy on shifting and undefined specs. In this context the file looks new, but it's not> We previously added a multilingual setting (now corresponding to the updated languages query point) but there was lack of clarity around handling and it was suspended. It has been determined that it *is* wanted, so there's no point deleting this yet as it can likely be repurposed.